definitions/scenarios/puppet.rb

```diff
@@ -0,0 +1,33 @@
+module ForemanMaintain::Scenarios
+  module Puppet
+    class DisablePuppet < ForemanMaintain::Scenario
+      metadata do
+        description 'Disable Puppet feature'
+        tags :puppet_disable
+        label :puppet_disable
+        param :remove_data, 'Purge the Puppet data after disabling plugin'
+        manual_detection
```

Comment:
This should be available on Satellite 7.0 only and not on capsules as well?

```suggestion
      metadata do
        description 'Disable Puppet feature'
        tags :puppet_disable
        label :puppet_disable
        param :remove_data, 'Purge the Puppet data after disabling plugin'
        confine do
          feature(:satellite) && feature(:satellite).current_minor_version == '7.0'
        end
        manual_detection
```

Reply:
Well actually no, we've ended up with a version, that works on capsules as well, so you can actually disable a puppet on capsules with this.

Reply:
> @upadhyeammit the whole enable/disable is designed to work on foreman/katello/proxy/proxy-content scenarios so I do not think we need the checks for sat. Tho foreman >= 3.0 would be appropriate, but not sure if that's necessary

The foreman >= 3.0 should be required as these commands will be available on 6.10 as well.